src/main/webapp/js/user-page-loader.js

```diff
@@ -110,9 +114,21 @@ function buildMessageDiv(message) {
   return messageDiv;
 }
 
+function buildLanguageLinks() {
+  const userPageUrl = '/user-page.html?user=' + parameterUsername;
+  const languagesListElement = document.getElementById('languages');
+  languagesListElement.appendChild(createListItem(createLink(userPageUrl + '&language=en', 'English')));
```

Comment:
ditto

Reply:
This is Javascript so I'm not sure if it's supposed to/will work as expected with the Java reformatter. What looks off about this chunk of code?